common/lib/calc/setup.py

```diff
@@ -7,6 +7,6 @@
     install_requires=[
         "pyparsing==2.0.1",
         "numpy",
-        "scipy"
+        "scipy<0.18"
```

Comment:
I had done that initially, however that introduced issues at run time if the versions specified didn't exactly match those required by ora2. I experimented with a range of combinations of versions and places that they were set and this was the only one that worked cleanly for me.

My thinking as to why I simpy did a loose requirement of `<0.18` is that version 0.18 is what introduced the build failure in the first place, so any version prior to that which is installed by the cascading requirements is what was already working. Essentially, "If it ain't broke, don't fix it" :)